lib/cf_deployer/driver/dry_run.rb

```diff
@@ -4,6 +4,30 @@ class DryRun
 
       @@enabled = false
 
+      def self.enabled?
+        @@enabled
+      end
+
+      def self.run_with_value value, &block
+        previous_value = @@enabled
+        @@enabled = value
+        begin
+          block.call
+        rescue => e
+          raise e
+        ensure
+          @@enabled = previous_value
+        end
+      end
+
+      def self.enable_for &block
```

Comment:
you added this `enable_for` code into the app just to be used by tests, right?
why would't you use rspec "allow" to handle setting the value of `self.enabled?`